sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClientCodeSnippets.java

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.data.tables;
+
+import com.azure.core.exception.HttpResponseException;
+import com.azure.data.tables.implementation.models.TableServiceErrorException;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class TableClientCodeSnippets {
+
+    public static void methods() {
+
+        //create a tableServiceClient
+        TableServiceClient tableServiceClient = new TableServiceClientBuilder()
+            .connectionString("connectionString")
+            .buildClient();
+
+        //create TableClient
+        TableClient tableClient = new TableClientBuilder()
```

Comment:
Should there be a way to get a TableClient from a TableServiceClient? I'm imagining something like:

```java
TableClient tableClient = tableServiceClient.queryTables("$selectString= TableName eq 'OfficeSupplies'")
                                            .first()
                                            .getClient();

```

(Or the same but with the hypothetical `getTable("TableName")` method I proposed in a previous comment.)